src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala

```diff
@@ -129,4 +129,14 @@ class BuiltinStringFunctionsTest
              ValString(""))))
   }
 
+  "An extract() function" should "return a list of strings matching a pattern" in {
```

Comment:
Please add an additional test for the case that the pattern is invalid. See the latest changes from master.